<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QualityCheckDlg</class>
 <widget class="QDialog" name="QualityCheckDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quailty Check Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="9">
      <widget class="QPushButton" name="pBtnExport">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="pBtnInput">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Browser...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="comboBoxScale">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>1:500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1:1000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1:2000</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="7">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" colspan="8">
      <widget class="QLineEdit" name="lineEditInput"/>
     </item>
     <item row="1" column="6">
      <widget class="QCheckBox" name="checkBoxExt">
       <property name="text">
        <string>Outdiffusion Extent</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QPushButton" name="pBtnCheck">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Check</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxFmt">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>GeoTiff/ Big Tiff/ Tiff (*.tif)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Erdas Imagine (*.img)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PCI Geomaticas (*.pix)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelInput">
       <property name="text">
        <string>Input Folder:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelFormat">
       <property name="text">
        <string>Filename Format:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Measuring Scale:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <column>
      <property name="text">
       <string>Filename</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Filename
Conventions</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Resolution
Correct</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rotation
Correct</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Southwest
Coordinate</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Dimensionality
Correct</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditInput</tabstop>
  <tabstop>pBtnInput</tabstop>
  <tabstop>comboBoxFmt</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
